projects/Mallard/src/App.tsx

```diff
@@ -158,7 +158,7 @@ export default class App extends React.Component<{}, {}> {
     componentDidMount() {
         SplashScreen.hide()
         weatherHider(apolloClient)
-        initialiseRemoteConfig()
+        remoteConfigService.init()
```

Comment:
I would have this at the top outside the `componentDidMount` to make sure it gets initialised as early as possible.

Reply:
Good shout, thank you.